src/index.js

```diff
@@ -1,7 +1,7 @@
 import { fireEvent } from "@testing-library/dom";
 
 function wait(time) {
-  return new Promise(function (resolve) {
+  return new Promise(function(resolve) {
```

Comment:
With all of these irrelevant formatting changes, it's really hard to grok the real diff.

Is this caused by the `pre-commit` hook, or is this a wonky editor plugin?

Reply:
Oh how embarrassing. Let me go through and figure out why I have whitespace changes.

Reply:
Circling back this is totally the pre-commit hook. Let me separate out the actual changes I proposed from the prettier changes.